shared.js

```diff
@@ -2,71 +2,72 @@
 // https://developer.mozilla.org/en-US/docs/Web/API/AudioContext
 let audioContext = new AudioContext();
 
-// Generates frequency to oscillate based on note and octave
-// https://gist.github.com/marcgg/94e97def0e8694f906443ed5262e9cbb?permalink_comment_id=3896533#gistcomment-3896533
-var NOTES = ["C", "C#", "D", "Eb", "E", "F", "F#", "G", "G#", "A", "Bb", "B"];
-function getPitch(note, octave) {
-    var step = NOTES.indexOf(note);
-    var power = Math.pow(2, (octave * 12 + step - 57)/12);
-    var pitch = 440 * power;
-    return pitch;
-}
-
-function playInstrumentNote(instrument, note) {
-    // Get the base note and the octave so that I can get the frequency to oscillate
+// Number of semitones each note is above C
+const noteToSemitones = {
```

Comment:
Shouldnt have removed NOTES. If you want the index, you could have us|d `NOTES.indexOf('C#')`.

Reply:
Ok I see that you need A# instead of Bb. So nevermind. I do want to keep the old code though, so is this replacing the code that synthetically create sound?